deliverables/README.md

```diff
@@ -6,67 +6,193 @@ In order to deliver homework consisting of questions and answers, just edit this
+a list of structured text with headings, images, links etc. What HTML does is that it frames these structures
+and produces a raw representation of our defined page. Later on, Cascading Style Sheets(CSS) can be applied 
+for styling, and Javascript for interactivity.
+
+**What is the role of CSS in a web page?**  
+CSS is used to control a page's style or look. The overall appearance of a page we are used to seeing online is different from how a simple html file would appear. That is because CSS brings to the table the style aspect 
+of web design, which, when used, makes our structured html file look dynamic and fluid.
+
+**What is the role of Javascript in a web page?**  
+Javascript is used to make a web page interactive.
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### Meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?**  
+Modularization, or in this case the separation of HTML and CSS, allow for efficiency, 
+maintainability and reusability. This separation allows for low coupling, making updating or debugging 
+our file much easier since only one part has to be dealt with, and it is easier to identify and modify.
+On the same note, we can reuse our code much more easily when they are separated.
+
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?
+**What are classes and IDs (and how are they different)?**  
+Classes are HTML attributes used to assign one or more subtypes to an element of a certain type.
+IDs are attributes used to provide unique identifiers to elements within a document. The difference 
+between them comes in that an ID has to be absolutely unique for an element within a page, whereas the
+class attribute may be used to apply the same value to multiple elements in a page.
+
+**What are elements?**  
+Much like an atom that is made up of three sub-atomic particles consisting of protons, electrons and neutrons, 
+similarly, an HTML element is made up of a opening tag, closing tag, and text in between. Elements are
+the atomic building blocks of our HTML code.
+
+**What are tags?**  
+Tags are part of HTML elements that define the meaning and structure of our HTML code. They most often come
+in pairs, namely start and closing tags. Some tags may define a table within our code, some a heading,
+and we even define an html file using starting and closing html tags (<"html"></"html">).
+Sometimes tags can be used for styling (eg. <"strong"> [text] </"strong">, makes our [text] bold etc.), but
+for the most part, we leave the styling aspect to CSS.
+
+**What are attributes?**   
+HTML attributes are used to define different characteristics of elements. Attributes are entered within 
+opening tags, and they affect the behaviour of whatever is in between the opening and closing tags
+(eg. that can be a text, image etc). Most attributes have the following form: a type that initializes
```

Comment:
Some attributes have no value - their existence is itself the message to the browser.  For example  

`<input type="checkbox" checked />`  

This may not be 100% valid html5 syntax (or it could be, I don't know), but browsers will treat that the same as if you had put  

`<input type="checkbox" checked="checked" />`